music_album.rb

```diff
@@ -0,0 +1,37 @@
+require 'json'
+require_relative './item'
+require_relative './label'
+require_relative './genre'
+
+class MusicAlbum < Item
+  attr_accessor :title, :on_spotify
+
+  def initialize(*args, on_spotify)
+    super(*args)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super || @on_spotify
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md#group-task:~:text=should%20return%20true%20if%20parent%27s%20method%20returns%20true%20AND%20if%20on_spotify%20equals%20true), Please make return true if parent's method returns true `AND` if on_spotify equals true.